.github/workflows/tf_apply_production.yml

```diff
@@ -52,3 +52,7 @@ jobs:
       - name: Apply api
         working-directory: terragrunt/env/production/api
         run: terragrunt apply --terragrunt-non-interactive -auto-approve
+
```

Comment:
Could you move this step to before the `Apply api` step since the API's Lambda function it will depend on it.

Reply:
Done! Thank you for bringing this up - should have realized this...